cogboard-app/src/test/kotlin/com/cognifide/cogboard/config/EndpointTest.kt

```diff
@@ -1,54 +1,56 @@
 package com.cognifide.cogboard.config
 
-import io.vertx.core.json.JsonArray
 import io.vertx.core.json.JsonObject
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.BeforeAll
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
 
```

Comment:
Tests updated. Added `@BeforeAll`  init method not to load endpoints every single time.